grpc-messenger-service/pom.xml

```diff
@@ -88,6 +98,25 @@
             <artifactId>commons-text</artifactId>
             <version>1.9</version>
         </dependency>
+        <dependency>
```

Comment:
why do you need these dependencies?

Reply:
Added for adding UTs which was not part of the PR as they still need some work. Will remove it for now.